### PR TITLE
vimPlugins: update

### PR DESCRIPTION
**Pull request by malob**
###### Description of changes

Update `vimPlugins` (ran `update.py` and updated hashes as required in overrides), revert d7bfa0dcc49491c7683a84e7edbf24b1c3bddcc6 which causes build issues since #178180 landed, and do a little cleanup in overrides.

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by malob** [comment]
Result of `nixpkgs-review pr 178520` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>8 packages marked as broken and skipped:</summary>
  <ul>
    <li>vimPlugins.The_NERD_tree</li>
    <li>vimPlugins.cmp-pandoc-nvim</li>
    <li>vimPlugins.command_T</li>
    <li>vimPlugins.fugitive</li>
    <li>vimPlugins.maktaba</li>
    <li>vimPlugins.rhubarb</li>
    <li>vimPlugins.snipmate</li>
    <li>vimPlugins.tlib</li>
  </ul>
</details>
<details>
  <summary>180 packages built:</summary>
  <ul>
    <li>spacevim</li>
    <li>vimPlugins.SchemaStore-nvim</li>
    <li>vimPlugins.SpaceVim</li>
    <li>vimPlugins.YouCompleteMe</li>
    <li>vimPlugins.aerial-nvim</li>
    <li>vimPlugins.ale</li>
    <li>vimPlugins.alpha-nvim</li>
    <li>vimPlugins.asyncrun-vim</li>
    <li>vimPlugins.aurora</li>
    <li>vimPlugins.auto-session</li>
    <li>vimPlugins.barbar-nvim</li>
    <li>vimPlugins.bufferline-nvim</li>
    <li>vimPlugins.catppuccin-nvim</li>
    <li>vimPlugins.catppuccin-vim</li>
    <li>vimPlugins.chad</li>
    <li>vimPlugins.cmp-buffer</li>
    <li>vimPlugins.cmp-clippy</li>
    <li>vimPlugins.cmp-copilot</li>
    <li>vimPlugins.cmp-dap</li>
    <li>vimPlugins.cmp-dictionary</li>
    <li>vimPlugins.cmp-digraphs</li>
    <li>vimPlugins.cmp-fish</li>
    <li>vimPlugins.cmp-fuzzy-buffer</li>
    <li>vimPlugins.cmp-fuzzy-path</li>
    <li>vimPlugins.cmp-git</li>
    <li>vimPlugins.cmp-greek</li>
    <li>vimPlugins.cmp-latex-symbols</li>
    <li>vimPlugins.cmp-look</li>
    <li>vimPlugins.cmp-neosnippet</li>
    <li>vimPlugins.cmp-npm</li>
    <li>vimPlugins.cmp-nvim-lsp-signature-help</li>
    <li>vimPlugins.cmp-nvim-tags</li>
    <li>vimPlugins.cmp-rg</li>
    <li>vimPlugins.cmp-snippy</li>
    <li>vimPlugins.cmp-tmux</li>
    <li>vimPlugins.cmp-vim-lsp</li>
    <li>vimPlugins.cmp-vimwiki-tags</li>
    <li>vimPlugins.cmp-zsh</li>
    <li>vimPlugins.coc-fzf</li>
    <li>vimPlugins.coc-lua</li>
    <li>vimPlugins.coc-nvim</li>
    <li>vimPlugins.command-t</li>
    <li>vimPlugins.comment-nvim</li>
    <li>vimPlugins.completion-treesitter</li>
    <li>vimPlugins.copilot-vim</li>
    <li>vimPlugins.coq-artifacts</li>
    <li>vimPlugins.coq-thirdparty</li>
    <li>vimPlugins.coq_nvim</li>
    <li>vimPlugins.dashboard-nvim</li>
    <li>vimPlugins.diffview-nvim</li>
    <li>vimPlugins.edge</li>
    <li>vimPlugins.editorconfig-vim</li>
    <li>vimPlugins.everforest</li>
    <li>vimPlugins.fern-vim</li>
    <li>vimPlugins.ferret</li>
    <li>vimPlugins.fidget-nvim</li>
    <li>vimPlugins.flutter-tools-nvim</li>
    <li>vimPlugins.fuzzy-nvim</li>
    <li>vimPlugins.fzf-lsp-nvim</li>
    <li>vimPlugins.fzf-lua</li>
    <li>vimPlugins.git-blame-nvim</li>
    <li>vimPlugins.gitsigns-nvim</li>
    <li>vimPlugins.glow-nvim</li>
    <li>vimPlugins.gruvbox-material</li>
    <li>vimPlugins.gruvbox-nvim</li>
    <li>vimPlugins.gv-vim</li>
    <li>vimPlugins.hologram-nvim</li>
    <li>vimPlugins.hop-nvim</li>
    <li>vimPlugins.hotpot-nvim</li>
    <li>vimPlugins.impatient-nvim</li>
    <li>vimPlugins.kanagawa-nvim</li>
    <li>vimPlugins.lazygit-nvim</li>
    <li>vimPlugins.lean-nvim</li>
    <li>vimPlugins.lexima-vim</li>
    <li>vimPlugins.lf-vim</li>
    <li>vimPlugins.lightspeed-nvim</li>
    <li>vimPlugins.litee-filetree-nvim</li>
    <li>vimPlugins.lsp_signature-nvim</li>
    <li>vimPlugins.lspcontainers-nvim</li>
    <li>vimPlugins.lspsaga-nvim</li>
    <li>vimPlugins.lualine-nvim</li>
    <li>vimPlugins.luasnip</li>
    <li>vimPlugins.lush-nvim</li>
    <li>vimPlugins.material-nvim</li>
    <li>vimPlugins.mini-nvim</li>
    <li>vimPlugins.minimap-vim</li>
    <li>vimPlugins.neogit</li>
    <li>vimPlugins.neorg</li>
    <li>vimPlugins.neoscroll-nvim</li>
    <li>vimPlugins.neovim-ayu</li>
    <li>vimPlugins.nerdtree</li>
    <li>vimPlugins.nui-nvim</li>
    <li>vimPlugins.null-ls-nvim</li>
    <li>vimPlugins.nvim-autopairs</li>
    <li>vimPlugins.nvim-bqf</li>
    <li>vimPlugins.nvim-cmp</li>
    <li>vimPlugins.nvim-dap</li>
    <li>vimPlugins.nvim-dap-ui</li>
    <li>vimPlugins.nvim-gdb</li>
    <li>vimPlugins.nvim-gps</li>
    <li>vimPlugins.nvim-hlslens</li>
    <li>vimPlugins.nvim-jdtls</li>
    <li>vimPlugins.nvim-lint</li>
    <li>vimPlugins.nvim-lspconfig</li>
    <li>vimPlugins.nvim-luapad</li>
    <li>vimPlugins.nvim-metals</li>
    <li>vimPlugins.nvim-notify</li>
    <li>vimPlugins.nvim-snippy</li>
    <li>vimPlugins.nvim-solarized-lua</li>
    <li>vimPlugins.nvim-tree-lua</li>
    <li>vimPlugins.nvim-treesitter</li>
    <li>vimPlugins.nvim-treesitter-context</li>
    <li>vimPlugins.nvim-ts-rainbow</li>
    <li>vimPlugins.octo-nvim</li>
    <li>vimPlugins.onedark-nvim</li>
    <li>vimPlugins.onedarkpro-nvim</li>
    <li>vimPlugins.orgmode</li>
    <li>vimPlugins.package-info-nvim</li>
    <li>vimPlugins.purescript-vim</li>
    <li>vimPlugins.refactoring-nvim</li>
    <li>vimPlugins.rose-pine</li>
    <li>vimPlugins.scrollbar-nvim</li>
    <li>vimPlugins.sonokai</li>
    <li>vimPlugins.space-vim</li>
    <li>vimPlugins.spellsitter-nvim</li>
    <li>vimPlugins.sqlite-lua</li>
    <li>vimPlugins.sved</li>
    <li>vimPlugins.tabline-nvim</li>
    <li>vimPlugins.telescope-cheat-nvim</li>
    <li>vimPlugins.telescope-frecency-nvim</li>
    <li>vimPlugins.telescope-fzf-native-nvim</li>
    <li>vimPlugins.telescope-fzf-writer-nvim</li>
    <li>vimPlugins.telescope-fzy-native-nvim</li>
    <li>vimPlugins.telescope-nvim</li>
    <li>vimPlugins.telescope-project-nvim</li>
    <li>vimPlugins.telescope-symbols-nvim</li>
    <li>vimPlugins.telescope-z-nvim</li>
    <li>vimPlugins.telescope-zoxide</li>
    <li>vimPlugins.tlib_vim</li>
    <li>vimPlugins.toggleterm-nvim</li>
    <li>vimPlugins.vifm-vim</li>
    <li>vimPlugins.vim-addon-actions</li>
    <li>vimPlugins.vim-addon-completion</li>
    <li>vimPlugins.vim-addon-goto-thing-at-cursor</li>
    <li>vimPlugins.vim-addon-nix</li>
    <li>vimPlugins.vim-addon-sql</li>
    <li>vimPlugins.vim-addon-syntax-checker</li>
    <li>vimPlugins.vim-addon-toggle-buffer</li>
    <li>vimPlugins.vim-autoformat</li>
    <li>vimPlugins.vim-bazel</li>
    <li>vimPlugins.vim-clap</li>
    <li>vimPlugins.vim-code-dark</li>
    <li>vimPlugins.vim-codefmt</li>
    <li>vimPlugins.vim-dadbod</li>
    <li>vimPlugins.vim-dadbod-completion</li>
    <li>vimPlugins.vim-dadbod-ui</li>
    <li>vimPlugins.vim-floaterm</li>
    <li>vimPlugins.vim-fugitive</li>
    <li>vimPlugins.vim-git</li>
    <li>vimPlugins.vim-ledger</li>
    <li>vimPlugins.vim-lsp</li>
    <li>vimPlugins.vim-maktaba</li>
    <li>vimPlugins.vim-markdown-composer</li>
    <li>vimPlugins.vim-mundo</li>
    <li>vimPlugins.vim-oscyank</li>
    <li>vimPlugins.vim-rhubarb</li>
    <li>vimPlugins.vim-sandwich</li>
    <li>vimPlugins.vim-slime</li>
    <li>vimPlugins.vim-smoothie</li>
    <li>vimPlugins.vim-snipmate</li>
    <li>vimPlugins.vim-snippets</li>
    <li>vimPlugins.vim-startuptime</li>
    <li>vimPlugins.vim-tpipeline</li>
    <li>vimPlugins.vim-ultest</li>
    <li>vimPlugins.vim-visual-multi</li>
    <li>vimPlugins.vim-which-key</li>
    <li>vimPlugins.vimspector</li>
    <li>vimPlugins.vimtex</li>
    <li>vimPlugins.winshift-nvim</li>
    <li>vimPlugins.xptemplate</li>
  </ul>
</details>

**Comment by malob** [comment]
Result of `nixpkgs-review pr 178520` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>7 packages marked as broken and skipped:</summary>
  <ul>
    <li>vimPlugins.The_NERD_tree</li>
    <li>vimPlugins.command_T</li>
    <li>vimPlugins.fugitive</li>
    <li>vimPlugins.maktaba</li>
    <li>vimPlugins.rhubarb</li>
    <li>vimPlugins.snipmate</li>
    <li>vimPlugins.tlib</li>
  </ul>
</details>
<details>
  <summary>181 packages built:</summary>
  <ul>
    <li>spacevim</li>
    <li>vimPlugins.SchemaStore-nvim</li>
    <li>vimPlugins.SpaceVim</li>
    <li>vimPlugins.YouCompleteMe</li>
    <li>vimPlugins.aerial-nvim</li>
    <li>vimPlugins.ale</li>
    <li>vimPlugins.alpha-nvim</li>
    <li>vimPlugins.asyncrun-vim</li>
    <li>vimPlugins.aurora</li>
    <li>vimPlugins.auto-session</li>
    <li>vimPlugins.barbar-nvim</li>
    <li>vimPlugins.bufferline-nvim</li>
    <li>vimPlugins.catppuccin-nvim</li>
    <li>vimPlugins.catppuccin-vim</li>
    <li>vimPlugins.chad</li>
    <li>vimPlugins.cmp-buffer</li>
    <li>vimPlugins.cmp-clippy</li>
    <li>vimPlugins.cmp-copilot</li>
    <li>vimPlugins.cmp-dap</li>
    <li>vimPlugins.cmp-dictionary</li>
    <li>vimPlugins.cmp-digraphs</li>
    <li>vimPlugins.cmp-fish</li>
    <li>vimPlugins.cmp-fuzzy-buffer</li>
    <li>vimPlugins.cmp-fuzzy-path</li>
    <li>vimPlugins.cmp-git</li>
    <li>vimPlugins.cmp-greek</li>
    <li>vimPlugins.cmp-latex-symbols</li>
    <li>vimPlugins.cmp-look</li>
    <li>vimPlugins.cmp-neosnippet</li>
    <li>vimPlugins.cmp-npm</li>
    <li>vimPlugins.cmp-nvim-lsp-signature-help</li>
    <li>vimPlugins.cmp-nvim-tags</li>
    <li>vimPlugins.cmp-pandoc-nvim</li>
    <li>vimPlugins.cmp-rg</li>
    <li>vimPlugins.cmp-snippy</li>
    <li>vimPlugins.cmp-tmux</li>
    <li>vimPlugins.cmp-vim-lsp</li>
    <li>vimPlugins.cmp-vimwiki-tags</li>
    <li>vimPlugins.cmp-zsh</li>
    <li>vimPlugins.coc-fzf</li>
    <li>vimPlugins.coc-lua</li>
    <li>vimPlugins.coc-nvim</li>
    <li>vimPlugins.command-t</li>
    <li>vimPlugins.comment-nvim</li>
    <li>vimPlugins.completion-treesitter</li>
    <li>vimPlugins.copilot-vim</li>
    <li>vimPlugins.coq-artifacts</li>
    <li>vimPlugins.coq-thirdparty</li>
    <li>vimPlugins.coq_nvim</li>
    <li>vimPlugins.dashboard-nvim</li>
    <li>vimPlugins.diffview-nvim</li>
    <li>vimPlugins.edge</li>
    <li>vimPlugins.editorconfig-vim</li>
    <li>vimPlugins.everforest</li>
    <li>vimPlugins.fern-vim</li>
    <li>vimPlugins.ferret</li>
    <li>vimPlugins.fidget-nvim</li>
    <li>vimPlugins.flutter-tools-nvim</li>
    <li>vimPlugins.fuzzy-nvim</li>
    <li>vimPlugins.fzf-lsp-nvim</li>
    <li>vimPlugins.fzf-lua</li>
    <li>vimPlugins.git-blame-nvim</li>
    <li>vimPlugins.gitsigns-nvim</li>
    <li>vimPlugins.glow-nvim</li>
    <li>vimPlugins.gruvbox-material</li>
    <li>vimPlugins.gruvbox-nvim</li>
    <li>vimPlugins.gv-vim</li>
    <li>vimPlugins.hologram-nvim</li>
    <li>vimPlugins.hop-nvim</li>
    <li>vimPlugins.hotpot-nvim</li>
    <li>vimPlugins.impatient-nvim</li>
    <li>vimPlugins.kanagawa-nvim</li>
    <li>vimPlugins.lazygit-nvim</li>
    <li>vimPlugins.lean-nvim</li>
    <li>vimPlugins.lexima-vim</li>
    <li>vimPlugins.lf-vim</li>
    <li>vimPlugins.lightspeed-nvim</li>
    <li>vimPlugins.litee-filetree-nvim</li>
    <li>vimPlugins.lsp_signature-nvim</li>
    <li>vimPlugins.lspcontainers-nvim</li>
    <li>vimPlugins.lspsaga-nvim</li>
    <li>vimPlugins.lualine-nvim</li>
    <li>vimPlugins.luasnip</li>
    <li>vimPlugins.lush-nvim</li>
    <li>vimPlugins.material-nvim</li>
    <li>vimPlugins.mini-nvim</li>
    <li>vimPlugins.minimap-vim</li>
    <li>vimPlugins.neogit</li>
    <li>vimPlugins.neorg</li>
    <li>vimPlugins.neoscroll-nvim</li>
    <li>vimPlugins.neovim-ayu</li>
    <li>vimPlugins.nerdtree</li>
    <li>vimPlugins.nui-nvim</li>
    <li>vimPlugins.null-ls-nvim</li>
    <li>vimPlugins.nvim-autopairs</li>
    <li>vimPlugins.nvim-bqf</li>
    <li>vimPlugins.nvim-cmp</li>
    <li>vimPlugins.nvim-dap</li>
    <li>vimPlugins.nvim-dap-ui</li>
    <li>vimPlugins.nvim-gdb</li>
    <li>vimPlugins.nvim-gps</li>
    <li>vimPlugins.nvim-hlslens</li>
    <li>vimPlugins.nvim-jdtls</li>
    <li>vimPlugins.nvim-lint</li>
    <li>vimPlugins.nvim-lspconfig</li>
    <li>vimPlugins.nvim-luapad</li>
    <li>vimPlugins.nvim-metals</li>
    <li>vimPlugins.nvim-notify</li>
    <li>vimPlugins.nvim-snippy</li>
    <li>vimPlugins.nvim-solarized-lua</li>
    <li>vimPlugins.nvim-tree-lua</li>
    <li>vimPlugins.nvim-treesitter</li>
    <li>vimPlugins.nvim-treesitter-context</li>
    <li>vimPlugins.nvim-ts-rainbow</li>
    <li>vimPlugins.octo-nvim</li>
    <li>vimPlugins.onedark-nvim</li>
    <li>vimPlugins.onedarkpro-nvim</li>
    <li>vimPlugins.orgmode</li>
    <li>vimPlugins.package-info-nvim</li>
    <li>vimPlugins.purescript-vim</li>
    <li>vimPlugins.refactoring-nvim</li>
    <li>vimPlugins.rose-pine</li>
    <li>vimPlugins.scrollbar-nvim</li>
    <li>vimPlugins.sonokai</li>
    <li>vimPlugins.space-vim</li>
    <li>vimPlugins.spellsitter-nvim</li>
    <li>vimPlugins.sqlite-lua</li>
    <li>vimPlugins.sved</li>
    <li>vimPlugins.tabline-nvim</li>
    <li>vimPlugins.telescope-cheat-nvim</li>
    <li>vimPlugins.telescope-frecency-nvim</li>
    <li>vimPlugins.telescope-fzf-native-nvim</li>
    <li>vimPlugins.telescope-fzf-writer-nvim</li>
    <li>vimPlugins.telescope-fzy-native-nvim</li>
    <li>vimPlugins.telescope-nvim</li>
    <li>vimPlugins.telescope-project-nvim</li>
    <li>vimPlugins.telescope-symbols-nvim</li>
    <li>vimPlugins.telescope-z-nvim</li>
    <li>vimPlugins.telescope-zoxide</li>
    <li>vimPlugins.tlib_vim</li>
    <li>vimPlugins.toggleterm-nvim</li>
    <li>vimPlugins.vifm-vim</li>
    <li>vimPlugins.vim-addon-actions</li>
    <li>vimPlugins.vim-addon-completion</li>
    <li>vimPlugins.vim-addon-goto-thing-at-cursor</li>
    <li>vimPlugins.vim-addon-nix</li>
    <li>vimPlugins.vim-addon-sql</li>
    <li>vimPlugins.vim-addon-syntax-checker</li>
    <li>vimPlugins.vim-addon-toggle-buffer</li>
    <li>vimPlugins.vim-autoformat</li>
    <li>vimPlugins.vim-bazel</li>
    <li>vimPlugins.vim-clap</li>
    <li>vimPlugins.vim-code-dark</li>
    <li>vimPlugins.vim-codefmt</li>
    <li>vimPlugins.vim-dadbod</li>
    <li>vimPlugins.vim-dadbod-completion</li>
    <li>vimPlugins.vim-dadbod-ui</li>
    <li>vimPlugins.vim-floaterm</li>
    <li>vimPlugins.vim-fugitive</li>
    <li>vimPlugins.vim-git</li>
    <li>vimPlugins.vim-ledger</li>
    <li>vimPlugins.vim-lsp</li>
    <li>vimPlugins.vim-maktaba</li>
    <li>vimPlugins.vim-markdown-composer</li>
    <li>vimPlugins.vim-mundo</li>
    <li>vimPlugins.vim-oscyank</li>
    <li>vimPlugins.vim-rhubarb</li>
    <li>vimPlugins.vim-sandwich</li>
    <li>vimPlugins.vim-slime</li>
    <li>vimPlugins.vim-smoothie</li>
    <li>vimPlugins.vim-snipmate</li>
    <li>vimPlugins.vim-snippets</li>
    <li>vimPlugins.vim-startuptime</li>
    <li>vimPlugins.vim-tpipeline</li>
    <li>vimPlugins.vim-ultest</li>
    <li>vimPlugins.vim-visual-multi</li>
    <li>vimPlugins.vim-which-key</li>
    <li>vimPlugins.vimspector</li>
    <li>vimPlugins.vimtex</li>
    <li>vimPlugins.winshift-nvim</li>
    <li>vimPlugins.xptemplate</li>
  </ul>
</details>

File: pkgs/applications/editors/vim/plugins/update.py [hunk 1]
```diff
@@ -116,13 +116,7 @@ def plugin2nix(self, pdesc: PluginDesc, plugin: pluginupdate.Plugin) -> str:
...
 
     global luaPlugins
-
-    # whitelist
-    luaPlugins = run_nix_expr(GET_PLUGINS_LUA) + [
-        "diffview-nvim",
-        "marks-nvim",
-        "nvim-biscuits"
-    ]
+    luaPlugins = run_nix_expr(GET_PLUGINS_LUA)
```
Comment (teto) [inline pkgs/applications/editors/vim/plugins/update.py:119]: I dont understand how it is still there, most likely I forgot to commit those removal back (they dont exist in my fork). Well spotted.

**Comment by teto** [comment]
why is vimPlugins.fugitive marked as broken ?

**Comment by malob** [comment]
> why is vimPlugins.fugitive marked as broken ?

@teto, I'm also confused by this, I think it's a quirk of `nixpkgs-review`, it's not broken, but for some reason it's skipping them.

I manually tested building all the plugins marked as broken or skipped, and they all build for me without issue except for `vimPlugins.cmp-pandoc-nvim` on `aarch64-darwin` (which makes sense since it depends on `python3Packages.pyopenssl` which is marked as broken on that system).

**Comment by teto** [comment]
@Mic92 any idea where this could come from ? I checked nixpkgs-review tracker and didn't find any relevant issue. I dont want to open an issue if this something I broke in the vimPlugins. Otherwise I can try looking into it

**Comment by teto** [comment]
I have a few pending changes for vim, let's merge this.

**Pull request merged**